test-variables.yaml.example

```diff
@@ -13,6 +13,18 @@ OCP_LDAP_USER:
   AUTH_TYPE: "ldap-provider-qe"
   USERNAME: ldap_user
   PASSWORD: abc123XYZ!
+OCP_LDAP_PROJECT_ADM:
+  AUTH_TYPE: "ldap-provider-qe"
+  USERNAME: ldap_user
+  PASSWORD: abc123XYZ!
+OCP_LDAP_SERVICE_ADM:
+  AUTH_TYPE: "ldap-provider-qe"
+  USERNAME: ldap_user
```

Comment:
Why do all the personas have the same username?

Reply:
@ikanias These fields needs to be updated by the user to represent the actuals